backend/wait_for_db.py

```
@@ -1,20 +1,22 @@
 import time
 import MySQLdb
+import os
 from django.db import connections
 from django.db.utils import OperationalError
 
+
 def wait_for_db():
```

Ruff (0.8.2)

Missing return type annotation for public function `wait_for_db`

Add return type annotation: `None`

(ANN201)